### PR TITLE
Add development guide

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
fixes: #325

### CODE REVIEW

**Comment by hzxuzhonghu** [comment]
Need help from @SrinivasChilveri

File: docs/development/development.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+
+## Cloning the code
+
+You will need to clone the main `volcano` repo to `$GOPATH/src/volcano.sh/volcano` for
+the below commands to work correctly.
+
+## Building the code
+
+To build volcano all components for your host architecture, go to
```
Comment (TommyLike) [inline docs/development/development.md]: where can I find the output binaries?

File: docs/development/development.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+```bash
+make vcctl
+make images
+make e2e-test-kind
+```
+
+If you want to run e2e test in a existing cluster with volcano deployed, run the following:
+
+```bash
+KUBECONFIG=${KUBECONFIG} go test ./test/e2e
```
Comment (TommyLike) [inline docs/development/development.md]: Need set the VK_BIN environment before running this go test command.

**Comment by SrinivasChilveri** [comment]
> Need help from @SrinivasChilveri

have added contribution, comminity member ship, trobulble shooting, getting started etc.

File: docs/development/development.md (hunk 2 above)
Comment (hzxuzhonghu) [reply docs/development/development.md]: ping @SrinivasChilveri

Comment (SrinivasChilveri) [reply docs/development/development.md]: Fixed

File: docs/development/development.md (hunk 1 above)
Comment (SrinivasChilveri) [reply docs/development/development.md]: changed

**Comment by hzxuzhonghu** [comment]
@TommyLike any other comments?

**Pull request merged**